app/src/main/java/com/example/composetemplate/di/RepositoriesModule.kt

```diff
@@ -1,14 +1,17 @@
 package com.example.composetemplate.di
 
+import com.example.composetemplate.repositories.AuthDataSource
 import com.example.composetemplate.repositories.AuthInteractor
 import com.example.composetemplate.repositories.FirebaseDataSource
 import com.example.composetemplate.repositories.LoginRepository
 import com.example.composetemplate.repositories.TestRepository
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
 import org.koin.dsl.module
 
 val repositoriesModule = module {
     factory { TestRepository(get(),get()) }
-    factory { AuthInteractor(get(),get()) }
-    factory { FirebaseDataSource() }
-    factory { LoginRepository(get()) }
+    factory { AuthInteractor(get()) }
```

Comment:
This is the repositoriesModule, and I see that we have three different types of classes here: 
Repository, Interactor, and DataSource. We should consider to create separate modules for each one or rename or add comments to clarify their purpose at the very least.

Reply:
agreed, I will do it in Tom missions